src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -33,6 +34,7 @@ class ConfirmationPage extends React.Component {
   }
 
   render() {
+    const signedIn = _get(this.props.saveInProgress, 'user.login.currentlyLoggedIn', false);
```

Comment:
Not sure how to get this value so I copied what's on the introduction page. LMK if this has to be done another way @jbalboni

Reply:
I think you'll need to add `user` to the `mapStateToProps` function below (it's at `state.user`) and get it from there.